app/views/admin/admin/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1>Petition admin</h1>
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <% action_counts do |counts| %>
+      <ul class="hub-actions grid-row">
+        <% counts.each do |state, count| %>
+          <li class="column-half action-<%= state.to_s.gsub('_', '-') %>">
```

Comment:
`<%= state.to_s.gsub('_', '-') %>` I wonder if we could refactor this out of the view. Maybe use the locale file to pull out the string with the state as a key, or add another variable 'css_class' to `generate_counts` and use it like

`counts.each do |state, css_class, count|`